### DISCUSSION TITLE
Support for RFC 9457

### DISCUSSION THREAD

**Discussion by adamralph** [projectdiscovery/httpx#1933] [General]
Has any consideration been given to supporting [RFC 9457](https://www.rfc-editor.org/rfc/rfc9457.html) (_Problem Details for HTTP APIs_)?

For example, some of the defined members in the response could be added to the error raised by `raise_for_status()`. Currently, if I get a 422, I log the error, but all I get is "422 Unprocessable Entity", but the detail I really want is in the [`detail` member](https://www.rfc-editor.org/rfc/rfc9457.html#section-3.1.4) of the response object.

**Comment by ehsandeep** [comment]
Correct project for this issue is https://github.com/encode/httpx
